### ISSUE TITLE
Error on first launch

### ISSUE THREAD

**Issue by svandragt** [svandragt/cuttlefish#92]
```
[Wed Jul  1 14:52:55 2020] PHP Fatal error:  Uncaught TypeError: Return value of Cuttlefish\Filesystem::requireFolder() must be of the type bool or null, none returned in /Users/sander/dev/cf/vendor/svandragt/cuttlefish/src/Filesystem.php:20
Stack trace:
#0 /Users/sander/dev/cf/vendor/svandragt/cuttlefish/src/Environment.php(49): Cuttlefish\Filesystem::requireFolder('../content/page...')
#1 /Users/sander/dev/cf/vendor/svandragt/cuttlefish/src/Environment.php(16): Cuttlefish\Environment->createSystemFolders()
#2 /Users/sander/dev/cf/vendor/svandragt/cuttlefish/src/App.php(25): Cuttlefish\Environment->__construct()
#3 /Users/sander/dev/cf/src/index.php(13): Cuttlefish\App->__construct('Cuttlefish\\Blog')
#4 {main}
  thrown in /Users/sander/dev/cf/vendor/svandragt/cuttlefish/src/Filesystem.php on line 20
[Wed Jul  1 14:52:55 2020] [::1]:58010 [200]: GET / - Uncaught TypeError: Return value of Cuttlefish\Filesystem::requireFolder() must be of the type bool or null, none returned in /Users/sander/dev/cf/vendor/svandragt/cuttlefish/src/Filesystem.php:20
Stack trace:
#0 /Users/sander/dev/cf/vendor/svandragt/cuttlefish/src/Environment.php(49): Cuttlefish\Filesystem::requireFolder('../content/page...')
#1 /Users/sander/dev/cf/vendor/svandragt/cuttlefish/src/Environment.php(16): Cuttlefish\Environment->createSystemFolders()
#2 /Users/sander/dev/cf/vendor/svandragt/cuttlefish/src/App.php(25): Cuttlefish\Environment->__construct()
#3 /Users/sander/dev/cf/src/index.php(13): Cuttlefish\App->__construct('Cuttlefish\\Blog')
#4 {main}
  thrown in /Users/sander/dev/cf/vendor/svandragt/cuttlefish/src/Filesystem.php on line 20
```
Suspect it is because cerain directories are not created, see `Cuttlefish\Environment->createSystemFolders()`

Workaround: reload the page.

**Comment by svandragt** [comment]
If the _cache directory is trashed then the request will never be successful because it's checking on the _logs folder. 😬

**Comment by svandragt** [comment]
Fixed by aa3ad37d0fe8bd35b6bd57ab7c0ed42c65cb9c51
